scripts/build.sh

```diff
@@ -314,6 +314,12 @@ case "$product-$method-$platform" in
         "${xcb_flags[@]}" \
         build \
         test
+
+    RunXcodebuild \
+        -workspace 'Firestore/Example/Firestore.xcworkspace' \
+        -scheme "Firestore_IntegrationTests_$platform" \
+        "${xcb_flags[@]}" \
+        build
     ;;
```

Comment:
Is adding test for a future PR?

Reply:
We've never had Firestore integration tests run on Travis. I haven't figured that out yet. When I do, I'll add to all the platforms we support.

Reply:
See https://github.com/firebase/firebase-ios-sdk/blob/master/scripts/install_prereqs.sh#L47